BraveWallet/Crypto/BuySendSwap/BuyTokenView.swift

```diff
@@ -75,41 +83,11 @@ struct BuyTokenView: View {
               .buttonStyle(BraveFilledButtonStyle(size: .normal))
               .frame(maxWidth: .infinity)
             }
-            .resetListHeaderStyle()
-            .listRowBackground(Color(.clear))
-          ) {
-          }
-          .listRowBackground(Color(.secondaryBraveGroupedBackground))
-        } else {
-          Section {
-            VStack(alignment: .leading, spacing: 4.0) {
-              Text(Strings.Wallet.buyTestTitle)
-                .font(.headline)
-              Text(String.localizedStringWithFormat(Strings.Wallet.buyTestDescription, networkStore.selectedChain.chainName))
-                .font(.subheadline)
-                .foregroundColor(Color(.secondaryBraveLabel))
-            }
-            .padding(.vertical, 6.0)
-            .listRowBackground(Color(.secondaryBraveGroupedBackground))
-          }
-          .listRowBackground(Color(.clear))
-          Section(
-            header:
-              Button(action: {
-                buyTokenStore.fetchTestFaucetUrl { urlString in
-                  guard let urlString = urlString, let url = URL(string: urlString) else {
-                    return
-                  }
-                  openWalletURL?(url)
-                }
-              }) {
-                Text(Strings.Wallet.buyTestButtonTitle)
-              }
-              .buttonStyle(BraveFilledButtonStyle(size: .normal))
-              .frame(maxWidth: .infinity)
```

Comment:
Did we intentionally remove buy functionality for Ropsten network? Maybe due to faucet being unavailable? Maybe we want to remove the unused localized strings if so.

Reply:
We no longer support eth testnets in Buy. Will remove related strings.